website/docs/cloud-docs/architectural-details/security-model.mdx

```diff
@@ -149,7 +149,11 @@ Workspaces may be configured to share their state with other workspaces within t
 
 ### Use separate agent pools for sensitive workspaces
 
-[Terraform Cloud Agents](/terraform/cloud-docs/agents) can be shared across all workspaces within an organization or [scoped to specific workspaces](/terraform/cloud-docs/agents#scope-an-agent-pool-to-specific-workspaces). If agent pools are shared among multiple workspaces, a malicious actor in one of those workspaces could exfiltrate the agent’s API token, access private resources from the perspective of the agent, or modify the agent’s environment, potentially impacting other workspaces. For this reason, we creating separate agent pools for sensitive workspaces and using the agent scoping setting to restrict which workspaces can target each agent pool.
+[Terraform Cloud Agents](/terraform/cloud-docs/agents) can be shared across all workspaces within an organization or [scoped to specific workspaces](/terraform/cloud-docs/agents#scope-an-agent-pool-to-specific-workspaces). If agent pools are shared among multiple workspaces, a malicious actor in one of those workspaces could exfiltrate the agent’s API token, access private resources from the perspective of the agent, or modify the agent’s environment, potentially impacting other workspaces. For this reason, we recommend creating separate agent pools for sensitive workspaces and using the agent scoping setting to restrict which workspaces can target each agent pool.
```

Comment:
Changed `For this reason, we creating separate...` to `For this reason, we recommend creating separate...`